draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -494,7 +498,8 @@ Definitions:
      messages are disclosed, R MUST evaluate to zero (0).
 - U, is the non-negative integer representing the number of undisclosed
      messages, i.e., U = L - R.
-- prf_len = ceil(ceil(log2(r))/8), where r defined by the ciphersuite.
+- expand_len = ceil((ceil(log2(r))+k)/8), where r and k are defined by
```

Comment:
Thanks for the feedback :D I did spend some time on that. The reason i decided to keep the `expand_len` term is because, as you noticed `hash_to_scalar` also uses it (with the name making much more sense there), and it seemed weird to have 2 different names for the same value.

I agree that this can be confusing though. However i don't think calling it `prf_out_len` is a better approach. Maybe `expand_scalar_len` or similar?? (open to suggestions, I'm really bad with names my self ;P)!